core/src/main/java/be/isach/ultracosmetics/cosmetics/gadgets/GadgetAntiGravity.java

```diff
@@ -48,7 +47,7 @@ protected void onRightClick() {
         portalParticles = ParticleDisplay.of(XParticle.PORTAL).offset(3, 3, 3).withCount(150).withLocation(as.getLocation());
         witchParticles = ParticleDisplay.of(XParticle.WITCH).offset(0.3, 0.3, 0.3).withCount(5).withLocation(as.getEyeLocation());
 
-        Bukkit.getScheduler().runTaskLater(getUltraCosmetics(), () -> running = false, 240);
+        getUltraCosmetics().getScheduler().runAtEntityLater(getPlayer(), () -> running = false, 240);
```

Comment:
This can probably be just `runLaterAsync` since it's just setting a variable

Reply:
Agreed, fixed!